src/main/scala/ai/privado/audit/AuditReportEntryPoint.scala

```diff
@@ -1,24 +1,74 @@
 package ai.privado.audit
 
-import ai.privado.cache.{RuleCache, TaggerCache}
+import ai.privado.cache.TaggerCache
+import ai.privado.exporter.JSONExporter
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
 import io.shiftleft.codepropertygraph.generated.Cpg
 import io.shiftleft.codepropertygraph.generated.nodes.ModuleDependency
-import org.apache.poi.ss.usermodel.{Cell, FillPatternType, IndexedColors, Row, Workbook}
+import org.apache.poi.ss.usermodel._
 import org.apache.poi.xssf.usermodel.{XSSFCellStyle, XSSFColor, XSSFWorkbook}
 
+import scala.collection.mutable.ListBuffer
 import scala.util.Try
 
 object AuditReportEntryPoint {
 
+  case class DataElementDiscoveryAudit(
+    className: String,
+    fileName: String,
+    filePriorityScore: String,
+    memberName: String,
+    memberType: String,
+    tagged: String,
+    sourceRuleId: String,
+    inputToCollection: String,
+    collectionEndpointPath: String,
+    collectionMethodFullName: String
+  )
+
+  implicit val DataElementDiscoveryAuditModelDecoder: Decoder[DataElementDiscoveryAudit] =
+    deriveDecoder[DataElementDiscoveryAudit]
+  implicit val DataElementDiscoveryAuditEncoder: Encoder[DataElementDiscoveryAudit] =
+    deriveEncoder[DataElementDiscoveryAudit]
+
+  def eliminateEmptyCellValueIfExist(str: String): String =
+    if (str == AuditReportConstants.AUDIT_EMPTY_CELL_VALUE) "" else str
+
+  def createDataElementDiscoveryJson(dataElementDiscoveryData: List[List[String]], repoPath: String) = {
+
+    val auditDataList = new ListBuffer[DataElementDiscoveryAudit]()
+    for (item <- dataElementDiscoveryData.drop(1)) {
+      auditDataList += DataElementDiscoveryAudit(
+        eliminateEmptyCellValueIfExist(item(0)),
+        eliminateEmptyCellValueIfExist(item(1)),
+        eliminateEmptyCellValueIfExist(item(2)),
+        eliminateEmptyCellValueIfExist(item(3)),
+        eliminateEmptyCellValueIfExist(item(4)),
+        eliminateEmptyCellValueIfExist(item(5)),
+        eliminateEmptyCellValueIfExist(item(6)),
+        eliminateEmptyCellValueIfExist(item(7)),
+        eliminateEmptyCellValueIfExist(item(8)),
+        eliminateEmptyCellValueIfExist(item(9))
+      )
+    }
+
+    JSONExporter.dataElementDiscoveryAuditFileExport("audit-sources.json", repoPath, auditDataList.toList)
```

Comment:
create a constant in audit Constants for the file name

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/606/commits/66666b5123cee6099681607bf440a6786a9166cc